devimint/Cargo.toml

```diff
@@ -31,7 +31,7 @@ nix = { version = "0.26.2", features = ["signal"] }
 rand = "0.8.5"
 serde_json = "1.0.94"
 tokio = { version = "1.26.0", features = ["full", "tracing"] }
-tonic_lnd = { git = "https://github.com/fedimint/tonic_lnd", branch="master", features = ["lightningrpc", "routerrpc"] }
+fedimint-tonic-lnd = { version = "0.1.0", features = ["lightningrpc", "routerrpc"] }
```

Comment:
Please use the renaming package trick I used for other crates.

Import as a renamed workspace dep here: https://github.com/fedimint/fedimint/blob/9c15acd60b715fccc5bf163941aca96f7c785832/Cargo.toml#L61 , then inherit where needed: https://github.com/fedimint/fedimint/commit/9826fc03c377c1699825fb5e87919f95691945a7#diff-3f39a723558ca0bac0086c38e83a005ec1cd87351875824049c98c37c2309be2R37